src/pgen/pgen.hpp

```diff
@@ -121,4 +121,9 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin);
 void Cleanup();
 } // namespace turbulence
 
+namespace rt {
+  using namespace parthenon::driver::prelude;
+  void ProblemGenerator(MeshBlock *pm, parthenon::ParameterInput *pin);
```

Comment:
```suggestion
  void ProblemGenerator(MeshBlock *pmb, parthenon::ParameterInput *pin);
```

For consistency. (`*pm` usually refers to the mesh pointer)